randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1473,18 +1437,28 @@ Extra - map_name: rm_a5b05
               # Suitless Method
               Morph Ball
               Any of the following:
+                  # Get past crumble blocks
                   Mid-Air Morph (Advanced)
                   Power Grip and Movement (Intermediate)
               Any of the following:
+                  # To climb the room
                   All of the following:
                       Walljump (Intermediate)
                       Power Grip or Mid-Air Morph (Advanced)
                   All of the following:
                       # IBJ
                       Infinite Bomb Jumping (Advanced) and Can Use Bombs
           All of the following:
-              # Intended Method
-              Gravity Suit and Can Use Spider Ball
+              # Gravity Suit Method
+              Gravity Suit
+              Any of the following:
+                  # Get past crumble blocks
+                  Can Use Spider Ball
+                  All of the following:
+                      Hi-Jump Boots and Movement (Intermediate)
+                      All of the following:
+                          # To climb the room
+                          Hijump Tunnel Climb
```

Comment:
This condition looks slightly weird and confusing.
If you want the HJ condition to get past the crumble blocks, you should also add midair morph or PG there for readability.